README.md

```diff
@@ -1,7 +1,30 @@
 # Microsoft Linux Package Repositories
-This repository is intended to communicate status of [packages.microsoft.com](https://packages.microsoft.com) and to receive feedback on the service. Issues can be filed in the [issues](https://github.com/microsoft/linux-package-reposotories/issues) page. For security issues, refer to [SECURITY.md](SECURITY.md).
 
-## Contributing
+## Overview 
+
+The Microsoft Linux Package Repositories are hosted on PMC ([packages.microsoft.com](https://packages.microsoft.com)) service. The PMC service is intended to support package hosting for customers with clients running a distribution of the Linux OS. Microsoft builds and supports a variety of software products for Linux systems and makes them available via standard APT and YUM package repositories.  
+
+## Configuring the repository on your Linux system 
+
+See how to [host/install/upgrade](https://docs.microsoft.com/en-us/windows-server/administration/linux-package-repository-for-microsoft-software) Microsoft's Linux software using your distribution's standard package management tools.  
+
+Microsoft's Linux Software Repository is comprised of multiple sub-repositories: 
+
+**prod** – The Production sub-repository is designated for packages intended to be used in production. These packages are commercially supported by Microsoft under the terms of the applicable support agreement or program that you have with Microsoft. 
```

Comment:
I would use plural here; there are multiple "prod" repositories (ubuntu, fedora, SUSE, etc). Perhaps clarify how the prod repositories are found (via top level folders: ubuntu/20.04/prod, or via their names: /repos/microsoft-ubuntu-focal-prod).